sympy/parsing/tests/test_latex.py

```diff
@@ -282,6 +282,11 @@ def test_parseable():
     for latex_str, sympy_expr in GOOD_PAIRS:
         assert parse_latex(latex_str) == sympy_expr, latex_str
 
+def test_parseable_lark():
+    from sympy.parsing.latex.lark import parse_latex_lark
+    for latex_str, sympy_expr in GOOD_PAIRS:
```

Comment:
what about:
```python
expected_failures = {4}
for i, (latex_str, sympy_expr) in enumerate(GOOD_PAIRS):
    if i in expected_failures:
        continue
    ...
```

Reply:
The `expected_failures` can be used to keep track of the failing examples.

Reply:
Done.